solr/solr-ref-guide/modules/query-guide/pages/terms-component.adoc

```diff
@@ -68,8 +68,57 @@ s|Required |Default: none
 +
 Specifies the field from which to retrieve terms.
 This parameter is required if `terms=true`.
+This parameter can be specified multiple times to retrieve terms for different fields.
```

Comment:
Can it also be comma separated? ie: `terms.fl=title,name`

Reply:
Oddly enough, it can NOT be..  "Error from server at null: undefined field: \"title,name\""     It would make MORE sense to me if that was the way it was defined.